### PR TITLE
Add support for App Domain tunnels

### PR DESCRIPTION
**Pull request by cwaeland**

### CODE REVIEW

File: dags/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/local/airflow/plugins/airflow-0.0.0-py3-none-any.whl
```
Comment (greg-compass) [inline dags/requirements.txt:1]: Do you really want a full path to a wheel file?

File: docker/Dockerfile [hunk 2]
```diff
@@ -31,6 +31,8 @@ COPY config/webserver_config.py ${AIRFLOW_USER_HOME}/webserver_config.py
 RUN chown -R airflow: ${AIRFLOW_USER_HOME}
 RUN chmod +x /entrypoint.sh
 
+RUN yum -y install socat
```
Comment (greg-compass) [inline docker/Dockerfile:34]: Is this going to leave a bunch of cruft in the yum cache? Maybe `yum -y install socat && yum clean all` if there's all the leftover RPM files in there after the build is attempted?

Comment (cwaeland) [reply docker/Dockerfile]: Updated.

File: dags/requirements.txt (hunk 1 above)
Comment (cwaeland) [reply dags/requirements.txt:1]: I do. This is purely for local development and unique to the MWAA environment. Developers will be providing that wheel in the plugins directory before building/starting the image.

**Review by greg-compass — APPROVED** [review]

**Pull request merged**